packages/integration-sdk-testing/src/__tests__/jest.test.ts

```diff
@@ -489,6 +489,7 @@ Find out more about JupiterOne schemas: https://github.com/JupiterOne/data-model
       platform: 'darwin',
       specialProp: 'abc',
       deviceId: 'udid of device',
+      lastSeenOn: 923618340000,
```

Comment:
`lastSeenOn` is now required on `Device` as [per this commit](https://github.com/JupiterOne/data-model/commit/c8a11465a287c2569fac875f7b2d079486c938e8)

💡 This is the exact date/time I was born